api/migrations/versions/349da0b8780a_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
```

Comment:
Please add a description for the migration

Reply:
Also please update the filename with some descriptive text after the underscore, so something like
349da0b8780a_journal_event_types.py